### PR TITLE
Remove pnpm-workspace since this is not a repo with multiple pkgs & update node and pnpm versions on build

### PR DESCRIPTION
**Pull request by anmolshres98**

### CODE REVIEW

**Comment by ben-polinsky** [comment]
LGTM

**Review by ben-polinsky — APPROVED** [review]

File: tests/fixtures/link.js [hunk 1]
```diff
@@ -28,5 +28,9 @@ function linkFixtureNoInternal() {
     "junction"
   );
 }
-
-linkFixtureNoInternal();
\ No newline at end of file
+try{
+  linkFixtureNoInternal();
```
Comment (paulius-valiunas) [inline tests/fixtures/link.js:32]: I would fix the spacing here

**Review by paulius-valiunas — APPROVED** [review]
Why was it there in the first place?

Anyway, I have a minor comment

**Comment by anmolshres98** [comment]
> Why was it there in the first place?
> 
> Anyway, I have a minor comment

Apparently when it was being migrated to here from the core repo, it was thought to be needed for the test to work but that is clearly not the case

File: tests/fixtures/link.js (hunk 1 above)
Comment (anmolshres98) [reply tests/fixtures/link.js:32]: it is done

**Pull request merged**